app/routes/_landing.($lang).docs/route.tsx

```diff
@@ -27,18 +31,44 @@ export const loader: LoaderFunction = async ({ params }) => {
   }
 }
 
+const replaceAll = (str: string, find: string, replace: string) => {
+  return str.replace(new RegExp(find, "g"), replace)
+}
+
 export default function DocsLayout() {
   const { data }: LoaderData = useLoaderData()
+  const [linksGroupItems, setLinksGroupItems] = useState<LinksGroupProps[]>([])
+
+  useEffect(() => {
+    if (data && data.length) {
+      console.log(data)
+      setLinksGroupItems(
+        data.map((doc: documentation) => {
+          return {
+            label: doc.slug ? replaceAll(doc.slug, "-", " ") : "", // We might want to choose a different property for the label
+            link: doc.slug ? doc.slug : "", // Assuming the slug can be used as the link
+            slug: doc.slug ? doc.slug : "",
+          }
+        }),
+      )
+    }
+  }, [data])
 
-  // Wrap with Layout/Sidenar
   return (
-    <div>
-      <ul>
-        {data.map((doc) => (
-          <li key={doc.id}>{doc && doc.translations && doc.translations[0]?.title}</li>
-        ))}
-      </ul>
-      <Outlet />
-    </div>
+    <Grid
+      gutter="md"
+      sx={{
```

Comment:
Just FYI... I have added Flex to blocks. Let's try to use it when possible in the future.